test/e2e/antreapolicy_test.go

```diff
@@ -144,6 +144,20 @@ func cleanupDefaultDenyNPs(k8s *KubernetesUtils, namespaces []string) error {
 	return nil
 }
 
+func testInvalidACNPNoPriority(t *testing.T) {
+	invalidNpErr := errors.New("invalid Antrea ClusterNetworkPolicy accepted")
+	builder := &ClusterNetworkPolicySpecBuilder{}
+	builder = builder.SetName("acnp-no-priority").SetAppliedToGroup(map[string]string{"pod": "a"}, nil, nil, nil)
+	np := metav1.Object{builder.Get()}
+	if acnp, ok := np.(*secv1alpha1.ClusterNetworkPolicy); ok {
+		log.Debugf("creating ACNP %v", acnp.Name)
+		_, err := k8sUtils.CreateOrUpdateCNP(acnp)
+		if err != nil {
```

Comment:
if `err == nil`?

Reply:
we want it to be errored no as validation should kick in and not allow this policy ?

Reply:
ah sorry.. my bad ..